recipes/default.rb

```diff
@@ -4,81 +4,54 @@
 #
 # Copyright (c) 2016 The Authors, All Rights Reserved.
 
-# install wget
-package 'wget' do
-  action :install
-end
-
-# install epel-release
-package 'epel-release' do
-  action :install
-end
+include_recipe 'delugeserver::yum_setup'
 
 # create system directories
 directory '/.downloads' do
   owner 'root'
   group 'root'
 end
 
-# download nux-dextop rpm
-execute 'nux-dextop_download' do
-  not_if do ::File.exists?('/.downloads/nux-dextop-release-0-5.el7.nux.noarch.rpm') end
-  command 'wget http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm -P /.downloads/'
-  action :run
-end
-
-# install nux-dextop
-package 'nux-dextop' do
-  source '/.downloads/nux-dextop-release-0-5.el7.nux.noarch.rpm'
-  action :install
-end
-
 # install deluge-daemon
-package 'deluge-daemon' do
+package 'deluge' do
   action :install
-  notifies :create, 'template[create_systemd_deluged_service]', :immediately
-end
-
-# install deluge-web
-package 'deluge-web' do
-  action :install
-end
-
-# create deluged.service file
-template 'create_systemd_deluged_service' do
-  action :create
-  path '/etc/systemd/system/deluged.service'
-  source 'deluged.service.erb'
-  owner 'root'
-  group 'root'
-  mode '0755'
-  notifies :restart, 'service[deluged]', :immediately
-end
-
-# create deluge-web.service file
-template 'create_systemd_deluge-web_service' do
-  action :create
-  path '/etc/systemd/system/deluge-web.service'
-  source 'deluge-web.service.erb'
-  owner 'root'
-  group 'root'
-  mode '0755'
-  notifies :restart, 'service[deluge-web]', :delayed
-end
-
-# deluge-web service
-service 'deluge-web' do
-  action :enable
+  version "#{node['deluge']['version']}.#{node['deluge']['release']}.nux"
+end
+
+if node['deluge']['logs']['enabled']
+  node.default['deluge']['daemon']['logs']['file'] = '-l /var/log/deluge/daemon.log'
+  node.default['deluge']['daemon']['logs']['level'] = '-L warning'
+  node.default['deluge']['web']['logs']['file'] = '-l /var/log/deluge/web.log'
+  node.default['deluge']['web']['logs']['level'] = '-L warning'
+  directory '/var/log/deluge' do
```

Comment:
I'd like to add this to the directories attributes and recipe but I'd have to extend it by accounting for mode...not the end of the world...I could do it in another feature branch down the road